packages/githublight/README.md

```diff
@@ -0,0 +1,40 @@
+# Github Light
+
+A theme that mimics github light theme. It is heavily inspired in the `github light theme default` for VSCode available [here](https://github.com/primer/github-vscode-theme).
+
+## Usage ✨
+
+To enable the theme in JupyterLab: `Go to Settings -> JupyterLab Theme -> Github Light`
+
+## Colors
+
+All the colors used for text in the theme are compliant with [WCAG 2.1 standards](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html).
+
+Background color: ![##ffffff](https://via.placeholder.com/20/#ffffff/#ffffff.png) `#ffffff`
+
+Highlight color: ![#321f7ad9](https://via.placeholder.com/20/321f7ad9/321f7ad9.png) `#321f7ad9`
+
+**WCAG compliance**
```

Comment:
I'm not sure how to read this table. The contrast values in the table are for each color against a white (#fff) background, correct? 

Should the last two columns reversed? I would have thought that the second row, for the color #6e7781, would be AA for normal text, but AAA for large.

Reply:
Agree - the table or some text should specify which colour combinations are being displayed (white/colours or highlight/colours).

+1 on the columns seeming reversed